pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -328,6 +328,19 @@ func (o *Options) Run() error {
 			fmt.Fprintf(o.Out, "Upgradeable=False\n\n  Reason: %s\n  Message: %s\n\n", c.Reason, c.Message)
 		}
 
+		if cv.Spec.Channel != "" {
+			if cv.Spec.Upstream == "" {
+				fmt.Fprint(o.Out, "Upstream is unset, so the cluster will use an appropriate default.\n")
```

Comment:
s/an appropriate default/ the default/ . Because we do not have multiple upstream options.